packages/web/app/package.json

```
@@ -5,7 +5,8 @@
   "scripts": {
     "dev": "next dev",
     "start": "next start",
-    "build": "bob runify --single"
+    "build": "bob runify --single",
+    "check:ts": "tsc"
```

You can do `"typecheck": "tsc --noEmit"` and it will run as part of CI

---

```suggestion
    "typecheck": "tsc"
```

--noEmit is not necessary as it is already set within the tsconfig.json

---

okok